src/js/pages/Arbitrators/index.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import {withRouter} from "react-router-dom";
+import { ListGroup, ListGroupItem, Button, Row,  Col } from 'reactstrap';
+import classnames from 'classnames';
+import metadata from '../../features/metadata';
+import network from '../../features/network';
+import arbitration from '../../features/arbitration';
+import { addressCompare } from '../../utils/address';
+import Loading from '../../components/Loading';
+import ErrorInformation from '../../components/ErrorInformation';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+class Arbitrators extends Component {
+  constructor(props) {
+    super(props);
+    props.getArbitrators(props.address, true);
+    this.loadedUsers = [];
+
+  }
+
+  componentDidUpdate(prevProps) {
+    if ((!prevProps.arbitrators && this.props.arbitrators) || prevProps.arbitrators.length !== this.props.arbitrators.length || Object.keys(this.props.users).length !== this.props.arbitrators.length) {
```

Comment:
Not sure the last part is ok. ` Object.keys(this.props.users).length !== this.props.arbitrators.length`
I think it should be ` Object.keys(this.props.users).length < this.props.arbitrators.length`
because there will most likely be more users than arbitrators

Reply:
good catch!